extends "res://content/shared/VideoTutorial.gd"

func _ready():
	$PanelContainer/MarginContainer/VBoxContainer/HBoxContainer/Text3/VBoxContainer/HBoxContainer/LabelDesc3_1.text = tr("level.tutorial.excavator.carry.desc11")
	$PanelContainer/MarginContainer/VBoxContainer/HBoxContainer/Text3/VBoxContainer/LabelDesc3_2.text = tr("level.tutorial.excavator.carry.desc12")
	$PanelContainer/MarginContainer/VBoxContainer/HBoxContainer/Text3/VBoxContainer2/HBoxContainer2/LabelDesc4_1.text = tr("level.tutorial.excavator.carry.desc21")
	$PanelContainer/MarginContainer/VBoxContainer/HBoxContainer/Text3/VBoxContainer2/LabelDesc4_2.text = tr("level.tutorial.excavator.carry.desc22")
	$PanelContainer/MarginContainer/VBoxContainer/HBoxContainer/Text3/VBoxContainer3/HBoxContainer2/LabelDesc5_1.text = tr("level.tutorial.excavator.carry.desc31")
	$PanelContainer/MarginContainer/VBoxContainer/HBoxContainer/Text3/VBoxContainer3/LabelDesc6_2.text = tr("level.tutorial.excavator.carry.desc32")

	super._ready()
	
